src/Http/Routes/Configuration/Access.php

```diff
@@ -40,6 +40,11 @@
     'uses' => 'AccessController@editRole',
 ]);
 
+Route::post('/roles/edit/{id}', [
+    'as'   => 'configuration.access.roles.rename',
+    'uses' => 'AccessController@renameRole',
+])->where('id', '[0-9]+');
```

Comment:
i have the feeling that we have avoided verification inside the routes file. Personally i would have created a verification for the request.

Reply:
Indead, the bouncer is set into the parent group https://github.com/warlof/web/blob/b99ee2e0ceb2d04d5a4b595b0ac2225d29b8b979/src/Http/routes.php#L126

Reply:
i didn't meant bouncer, i meant verification: https://laravel.com/docs/5.5/validation#creating-form-requests

Reply:
This is a simple constraint for a URL parameter that is fine (and quite nice actually).